.changeset/rich-pigs-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/react": minor
+---
+
+[ReactSDK] update useClaimedNFTs to only fetch claimed NFTs
```

Comment:
Don't need the "[ReactSDK]" here, the changelog already applies to this package so that's automatic, we just do that on the PR titles themselves :)